resources/services/lambda/functions.go

```diff
@@ -416,85 +517,79 @@ func LambdaFunctions() *schema.Table {
 						Type:        schema.TypeJSON,
-						Resolver:    schema.PathResolver("RoutingConfig.AdditionalVersionWeights"),
+						Resolver:    schema.PathResolver("AliasConfiguration.RoutingConfig.AdditionalVersionWeights"),
 					},
-				},
-			},
-			{
-				Name:          "aws_lambda_function_event_invoke_configs",
-				Description:   "A configuration object that specifies the destination of an event after Lambda processes it. ",
-				Resolver:      fetchLambdaFunctionEventInvokeConfigs,
-				IgnoreInTests: true,
-				Columns: []schema.Column{
 					{
-						Name:        "function_cq_id",
-						Description: "Unique CloudQuery ID of aws_lambda_functions table (FK)",
-						Type:        schema.TypeUUID,
-						Resolver:    schema.ParentIdResolver,
+						Name:        "url_config_auth_type",
+						Description: "The type of authentication that your function URL uses",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("UrlConfig.AuthType"),
 					},
 					{
-						Name:        "on_failure_destination",
-						Description: "The Amazon Resource Name (ARN) of the destination resource.",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("DestinationConfig.OnFailure.Destination"),
+						Name:        "url_config_creation_time",
+						Description: "When the function URL was created, in ISO-8601 format (https://www.w3.org/TR/NOTE-datetime) (YYYY-MM-DDThh:mm:ss.sTZD).  This member is required.",
+						Type:        schema.TypeTimestamp,
+						Resolver:    client.ISODateResolver("UrlConfig.CreationTime"),
 					},
 					{
-						Name:        "on_success_destination",
-						Description: "The Amazon Resource Name (ARN) of the destination resource.",
+						Name:        "url_config_function_arn",
+						Description: "The Amazon Resource Name (ARN) of your function.  This member is required.",
 						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("DestinationConfig.OnSuccess.Destination"),
+						Resolver:    schema.PathResolver("UrlConfig.FunctionArn"),
 					},
 					{
-						Name:        "function_arn",
-						Description: "The Amazon Resource Name (ARN) of the function.",
+						Name:        "url_config_function_url",
+						Description: "The HTTP URL endpoint for your function.  This member is required.",
 						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("UrlConfig.FunctionUrl"),
 					},
 					{
-						Name:        "last_modified",
-						Description: "The date and time that the configuration was last updated.",
+						Name:        "url_config_last_modified_time",
+						Description: "When the function URL configuration was last updated, in ISO-8601 format (https://www.w3.org/TR/NOTE-datetime) (YYYY-MM-DDThh:mm:ss.sTZD).  This member is required.",
 						Type:        schema.TypeTimestamp,
+						Resolver:    client.ISODateResolver("UrlConfig.LastModifiedTime"),
```

Comment:
Love this addition!

Reply:
Thanks. I've crated an issue (cloudquery/cq-provider-sdk#223) to crate a generic implementation of this resolver to make it usable by all the providers